packages/Telephony/Readme.md

```diff
@@ -120,19 +120,22 @@ The Stop Recording action stops recording of the conversation. Note that it is n
 ## **Aggregate DTMF Input (n)**
 Prompts the user for multiple inputs that are aggregated until a specified character length is met or exceeded.
 Speech, DTMF inputs, and chat provided characters can all be used to provide input, but any inputs that aren't the characters 1,2,3,4,5,6,7,8,9,0,#,*, or some combination of said characters are dropped.
+In case the timeout in milliseconds is reached, the dialog will end and show the Default Value property to the user.
```

Comment:
`"Show the Default Value property to the user"` could be reworded better to indicate that the Default Value property will be spoken to the user, if that's the intent.

Reply:
Hi Steven, sure!. As requested, we will be doing additional changes to this PR. If you want I can let you know as soon as we pushed them.

Reply:
Hey @luislera, I synced with Jim and agree with his requested changes, please disregard my comment for now and perform his changes. After his reqs are complete, I'll take another look at the PR ☺️